e2e/challenges_test.go

```diff
@@ -198,6 +198,37 @@ func TestChallengeTLS_Run_Revoke_Non_ASCII(t *testing.T) {
 	}
 }
 
+func TestRegistrar_UpdateAccount(t *testing.T) {
```

Comment:
One drive-by suggestion: This unit test might also benefit from a test case that verifies **removing** all contact addresses with an `UpdateRegistration` operation works as expected.

Reply:
Removing contact addresses is not supported at the moment, I wish to add it later but it will change the User interface.